packages/documentation/.storybook/blocks/footer.tsx

```diff
@@ -83,7 +83,7 @@ export default (params: { pathToStoryFile?: String }) => (
       </div>
 
       <div className="container d-flex justify-content-between py-regular">
-        <strong>&copy; 2022 Swiss Post Ltd.</strong>
+        <strong>&copy; 2024 Swiss Post Ltd.</strong>
```

Comment:
```suggestion
        <strong>&copy; {new Date().getFullYear()} Swiss Post Ltd.</strong>
```

Seems even more cumbersome to do this every year by hand 😄!

Reply:
Forget that we are working with tsx and mdx for the doc (markdown is in my head). Good!